packages/server/src/provider/provider.ts

```diff
@@ -57,4 +65,12 @@ export interface Provider extends CommonProvider<ServerProviderStatus> {
     context: EvaluationContext,
     logger: Logger,
   ): Promise<ResolutionDetails<T>>;
+
+  /**
+   * Track a thing
+   * @param occurrenceKey
+   * @param context
+   * @param occurrenceDetails
```

Comment:
Are we doing these later?

Reply:
https://github.com/open-feature/js-sdk/pull/1020#discussion_r1779083185